packages/core/src/App/Containers/QuestionnaireModal/questionnarie-modal.scss

```diff
@@ -0,0 +1,62 @@
+.dc-modal__container_questionnarie-modal {
+    @include mobile {
+        width: 100vw !important;
+    }
+}
+
+.questionnarie-modal {
+    max-inline-size: 44rem;
+    border-radius: 1.6rem;
+    padding-block: 3.8rem !important;
```

Comment:
please don't use !important. instead, please reorder classnames, or add this style in the exact classname where it's needed.

Reply:
done by adding new selector 👍